examples/naming-conventions/src/naming-conventions.idl

```diff
@@ -0,0 +1,13 @@
+
+namespace naming_conventions {
+  snake_case_object snake_case_method(u32 id);
+  CamelCaseObject camelCaseMethod(u32 id);
```

Comment:
Interesting philosophical point here about what to allow in the IDL, and I'm kind of in two minds about it. On one hand, it seems wrong to outright error out if something declared in this file does not match a particular naming convention, but on the other hand, allowing mixing conventions like this might lead to confusion. Perhaps it's the sort of thing that might one day be a candidate for a warning.

Reply:
See discussion below. I think the chance of collision is too high here.